sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_scalar_aggregates_v1.sql.py

```diff
@@ -96,7 +96,8 @@ def generate_sql(
                 app_version,
                 app_build_id,
                 channel,
-                {aggregates}
+                {aggregates},
+                max(profile_group_id) AS profile_group_id
```

Comment:
I know that as a rule of thumb it's better to add new columns at the end, but since this generated query gets rather big, can we place `profile_group_id` right under under `client_id`? I think it'll improve the readability of the generated query by a bunch.
I'll leave it up to you, but if you decide to do it, you have my sign-off that I'll deal with any catastrophic event that might happen as a result of it.
Same for the other occurrences.

Reply:
Of course I can move it under client ID.  @edugfilho Did you prefer I group by profile group ID and not take an aggregate? I was originally avoiding that because I didn't want downstream things to be impacted but now I'm debating if I should just include it as a group by field

Reply:
That's a good consideration. I don't think there will be any impact, so I'd vote for changing the grain here. Also available for dealing with potential fallout.

Reply:
OK then I will re-write these to include it in the group by, if there is no downstream impact, I prefer that approach as well.